Formula/objconv.rb

```diff
@@ -0,0 +1,53 @@
+class Objconv < Formula
+  desc "Object file converter"
+  homepage "https://www.agner.org/optimize/#objconv"
+  url "https://www.agner.org/optimize/objconv.zip"
+  version "2022-08-17"
+  sha256 "024aa73b54022ae6ab19e4363b6b8e3654351a524f369e2090cd5cce5ae541f0"
+  license "GPL-3.0-or-later"
+
+  livecheck do
+    url :homepage
+    regex(/objconv.*last\s+modified:\s+(\d{4}-(?:[A-Z][a-z]{2}|\d{2})+-\d{2})/i)
+    strategy :page_match do |page, regex|
+      date = page.scan(regex).flatten.first
```

Comment:
```suggestion
    regex(/objconv\.zip,.*?last\s+modified:\s+(\d{4}-(?:[a-z]+|\d{2})-\d{2})/im)
    strategy :page_match do |page, regex|
      date = page[regex, 1]
```

You have the general idea but we may want to make a few tweaks to the regex:

* This replaces `objconv.*` with `objconv.zip,.*?`, as the additional specificity is necessary to avoid matching the "last modified" date from other files (since `.*` is very loose). The original regex works because the `strategy` block only uses the first match but it could break if we're not careful, so there's something to be said for being a little more specific in this case. There's probably further room for improvement here but this works for now.
* Since this is a case-insensitive regex, `[A-Z][a-z]{2}` isn't very meaningful. We could use `[a-z]{3}` if we specifically want to target three letters but I'm not sure that would buy us much, so this suggestion simply uses `[a-z]+`.
* This removes the `+` from the non-capturing group (i.e., the month), as it shouldn't repeat.
* This adds the multiline flag (`m`) to the regex, since we're matching loose text (i.e., we want this to continue to work if the HTML formatting changes with respect to line breaks).

This also replaces `page.scan(regex).flatten.first` with `page[regex, 1]`, which is the established way of only working with the first match on the page. Usually we want to work with all matches on a page but this is fine in this particular context. [If we did want to use the `map` approach in the future, it would be something like `page.scan(regex).map { |match| Date.parse(match.first).strftime("%Y-%m-%d") }`.]